openmc/summary.py

```diff
@@ -64,6 +71,26 @@ def nuclides(self):
         return self._nuclides
 
     @property
+    def fast_materials(self):
```

Comment:
Are you suggesting that, for example, this property should be turned into a `get_fast_materials()` method? I agree that the `fast_` names aren't desirable, I just put them in here as a placeholder.